src/app/app.module.ts

```diff
@@ -40,6 +42,8 @@ import { GenerateExamPageComponent } from './generate-exam-page/generate-exam-pa
 import { SearchForAnExamPageComponent } from './search-for-an-exam-page/search-for-an-exam-page.component';
 import { CreateExamPageComponent } from './create-exam-page/create-exam-page.component';
 import { CreateQuestionPageComponent } from './create-question-page/create-question-page.component';
+import { CreateQuestionPageComponentPart1 } from './create-question-page/create-question-page-part1/create-question-page-part1.component';
```

Comment:
A better name would be `CreateQuestionPageComponent.QuestionSection`... something more descriptive, `..Part1`. Nobody who comes behind you working on this code will know what `..Part1` means.